tableaudocumentapi/connection.py

```diff
@@ -26,6 +26,11 @@ def __init__(self, connxml):
         self._dbname = connxml.get('dbname')
         self._server = connxml.get('server')
         self._username = connxml.get('username')
+        self._authentication = connxml.get('authentication')
+        self._class = connxml.get('class')
+
+    def __repr__(self):
```

Comment:
The repr is mainly for troubleshooting and printing to stdout -- it's common to enclose it in angle brackets and include a memory address -- here I just went with something that was 'not real' enough to notice that it's a debug thing that I'm printing but enough to tell me it's the right object.

It could be:
`"<Connection server={} dbname={} class={} />` which is probably usually enough to note if it's the right connection or not.